repository/src/main/java/org/alfresco/repo/rule/RuleServiceImpl.java

```diff
@@ -783,7 +783,9 @@ public Rule saveRule(NodeRef nodeRef, Rule rule)
             }
 
             // Update the properties of the rule
-            this.nodeService.setProperty(ruleNodeRef, ContentModel.PROP_TITLE, rule.getTitle());
+            String title = rule.getTitle();
+            ParameterCheck.mandatoryString("Rule name", title);

```

Comment:
Does V0 API check for rule name or allow empty ?  I assume there are no custom integrations (eg. using V0) that don't supply a name. 

We may need to (eventually) remember to list this change in ACS 7.3.0 release notes ?

Reply:
Yes - weirdly there is validation in both the v0 webscript and in Share, but there was nothing in the service layer.